src/content/ko/fundamentals/performance/critical-rendering-path/constructing-the-object-model.md

```diff
@@ -29,22 +28,22 @@ description: 브라우저가 DOM 및 CSSOM 트리를 생성하는 방법에 대
 
 가장 단순한 경우인 몇몇 텍스트와 하나의 이미지만 포함하는 일반 HTML 페이지부터 살펴보도록 하겠습니다. 브라우저가 이 페이지를 어떻게 처리하나요?
 
-<img src="images/full-process.png" alt="DOM 생성 프로세스">
+<img src="images/full-process.png" alt="DOM construction process">
 
 1. **변환:** 브라우저가 HTML의 원시 바이트를 디스크나 네트워크에서 읽어와서, 해당 파일에 대해 지정된 인코딩(예: UTF-8)에 따라 개별 문자로 변환합니다.
-1. **토큰화:** 브라우저가 문자열을 [W3C HTML5 표준](http://www.w3.org/TR/html5/){: .external }에 지정된 고유 토큰으로 변환합니다(예: '&lt;html&gt;', '&lt;body&gt;' 및 꺽쇠괄호로 묶인 기타 문자열). 각 토큰은 특별한 의미와 고유한 규칙을 가집니다.
-1. **렉싱:** 방출된 토큰은 해당 속성 및 규칙을 정의하는 '객체'로 변환됩니다.
-1. **DOM 생성:** 마지막으로, HTML 마크업이 여러 태그(일부 태그는 다른 태그 안에 포함되어 있음) 간의 관계를 정의하기 때문에 생성된 객체는 트리 데이터 구조 내에 연결됩니다. 이 트리 데이터 구조에는 원래 마크업에 정의된 상위-하위 관계도 포합됩니다. 즉, _HTML_ 객체는 _body_ 객체의 상위이고, _body_ 는 _paragraph_ 객체의 상위인 식입니다.
+2. **토큰화:** 브라우저가 문자열을 [W3C HTML5 표준](http://www.w3.org/TR/html5/){: .external }에 지정된 고유 토큰으로 변환합니다(예: '<html>', '<body>' 및 꺽쇠괄호로 묶인 기타 문자열). 각 토큰은 특별한 의미와 고유한 규칙을 가집니다.
```

Comment:
the HTML tags (html and body) need to be wrapped in backticks (`) otherwise they'll be properly rendered and break the view.
```
변환합니다(예: `<html>`, `<body>` 및 꺽쇠괄호로 묶인 기타 문자열).
```

Reply:
I guess GitLocalise had changed other parts of translations I didn't touch.
I'll check them, and fix.

Reply:
Closing this PR for appropriate update. :)